yb-voyager/src/tgtdb/suites/yugabytedbSuite.go

```diff
@@ -173,17 +173,8 @@ var YBValueConverterSuite = map[string]ConverterFn{
 		}
 		return string(hexValue), nil
 	},
-	"MAP": func(columnValue string, _ bool, _ *schemareg.ColumnSchema) (string, error) {
-		mapValue := make(map[string]interface{})
-		err := json.Unmarshal([]byte(columnValue), &mapValue)
-		if err != nil {
-			return columnValue, fmt.Errorf("parsing map: %v", err)
-		}
-		var transformedMapValue string
-		for key, value := range mapValue {
-			transformedMapValue = transformedMapValue + fmt.Sprintf("\"%s\"=>\"%s\",", key, value)
-		}
-		return fmt.Sprintf("'%s'", transformedMapValue[:len(transformedMapValue)-1]), nil //remove last comma and add quotes
+	"MAP": func(columnValue string, formatIfRequired bool, dbzmSchema *schemareg.ColumnSchema) (string, error) {
```

Comment:
- what about postgressuite? Do we re-use yugabytedbSuite? 
- we have some implementation for "MAP" in oracleSuite as well. I don't imagine that is useful, since we won't have MAP coming from oracle source at all, so remove it?

Reply:
> what about postgressuite? Do we re-use yugabytedbSuite?

yes we use the same for both

> we have some implementation for "MAP" in oracleSuite as well. I don't imagine that is useful, since we won't have MAP coming from oracle source at all, so remove it?

yeah ok